src/main/webapp/app/modules/login/LoginFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text } from '@adobe/react-spectrum'
+
+import { CommonConstants } from '../../shared/util/common.constants';
+
+// TODO: Test Cases
```

Comment:
Test cases will become mandatory from the next week, so please make sure these are done from next week!

Reply:
this design is not done yet..i have put questions to be answered ..once this design is ready..test cases will be written